src/components/Tabs/style.scss

```diff
@@ -43,3 +48,22 @@
     border-bottom: 1px solid $cGray600;
   }
 }
+
+.verticalTab {
```

Comment:
maybe vertical styles should just be an addition to "base" tab ones? seems strange that vertical would look inherently different than horizontal as far as things like background color of active tab, font size, etc.